make/configure.py

```diff
@@ -1535,6 +1535,7 @@ class Tools:
 
         xcodebuild = ToolProbe( 'XCODEBUILD.exe', 'xcodebuild', abort=False )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo', abort=False )
+        python     = ToolProbe( 'PYTHON.exe', os.path.basename(sys.executable), abort=True )
```

Comment:
I feel like this should follow a pattern more like https://github.com/HandBrake/HandBrake/blob/2766a279df69b7dde72582b7793ac39090d8683c/configure#L17

Otherwise, looks good. Thank you.